app/src/main/res/drawable/splash_image.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+  <item android:drawable="@color/color_palette_primary_color" />
```

Comment:
@TanishMoral11, Please change this to a component colour to fix the failing check. Please refer to the closed PR for a comment I made related to this.

Reply:
Hey @adhiamboperes, Thanks for the guidance! I've updated the color to a component color as recommended to address the failing check. Please let me know if there’s anything else needed!